src/lib/source/GeoJSONSource.ts

```diff
@@ -1,4 +1,4 @@
-import { GeoJSON, GeoJsonGeometryTypes, Feature } from 'geojson';
+import { GeoJSON, Feature, GeoJsonGeometryTypes } from 'geojson';
```

Comment:
It's curious how they use GeoJSON and GeoJson in the same lib... 😄

Reply:
yup, I thought the same... _why did they do that?_ :shrug: